core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -283,6 +283,17 @@ oppia.controller('CollectionPlayer', [
       $location.hash(id);
       $anchorScroll();
     };
+
+    $scope.closeOnClickingOutside = function($evt) {
+      explorationTile = $evt.currentTarget.childNodes[2];
+      console.log(explorationTile);
```

Comment:
Drop all console.logs before you submit a PR.

Reply:
Done